### ISSUE TITLE
 Re-design: Board general view

### ISSUE THREAD

**Issue by mquandalle** [wekan/wekan#94]
Re-design discussions:
* #126 For the home page and the activity feed
* [*current*] For the board general view
* #127 For the card detailed view

Current state of the board re-design
-----------

![current](https://cloud.githubusercontent.com/assets/1730702/5963339/825a59dc-a7eb-11e4-94b2-2c05182d0834.png) 

---

Hello,

Working with a free and open-source software allow us as a community to experiment with new features and new user interfaces. @yasaricli and I are both good developers, but I guess not excellent designers (and even if we were, input from the community are always invaluable).

I would like to open a brainstorming about the user interface and experience; feel free to share any UI/UX idea you have, and if you have a graphical talent also share some mock-ups of your ideas!

Let's start this thread by some picture of trello redesign tries:

![Trello redesign](https://cloud.githubusercontent.com/assets/1730702/5819476/75a6842e-a0be-11e4-99d7-bf676be8f8b5.png)

↑ by Fabrice Liut — http://www.liut.me/a-trello-redesign [1]

![trello_redesign_3](https://cloud.githubusercontent.com/assets/1730702/5819548/0ffed936-a0bf-11e4-8dab-4052edb09e03.jpg)

↑ by Duncan Malashock  — https://dribbble.com/shots/613931-Trello-Redesign-v3 [2]

![trello_full](https://cloud.githubusercontent.com/assets/1730702/5819605/61a751be-a0bf-11e4-8651-68368efbe1b8.png)

↑ by Stu Greenham — https://dribbble.com/shots/1854771-Trello-Redesign-Concept [3]

![trello-redesign-big](https://cloud.githubusercontent.com/assets/1730702/5819622/7e78eba4-a0bf-11e4-96a2-4c8e2a954844.png)

↑ by Fran Pérez  — https://dribbble.com/shots/540307-Trello-Redesign [4]

![desktop_hd](https://cloud.githubusercontent.com/assets/1730702/5819627/8cc231f2-a0bf-11e4-8e3c-7a8e884d9078.png)

↑ by Philippe HONG — https://dribbble.com/shots/1847564-Trello-Redesign [5]

And now some alternative kanban board softwares:

![swimlanes](https://cloud.githubusercontent.com/assets/1730702/5819743/5f14ab6c-a0c0-11e4-8e09-1c568cb502e7.png)

↑ https://kanbanflow.com/ [6]

![db6c44d9781d926d8890a8d0765eb788](https://cloud.githubusercontent.com/assets/1730702/5819766/87b58398-a0c0-11e4-944c-e1d8946ee4fb.png)

↑ https://www.odoo.com [7]

![selection_012](https://cloud.githubusercontent.com/assets/1730702/5819794/c2d5229e-a0c0-11e4-8892-34b8e8ee3d32.png)

↑ https://pintask.me [8]

![kanbanize](https://cloud.githubusercontent.com/assets/1730702/5841382/d03254de-a19c-11e4-923e-87c97590479c.jpg)

↑ https://kanbanize.com [9]

![board](https://cloud.githubusercontent.com/assets/1730702/5848073/cde1e34c-a1d4-11e4-9eb0-b323305128ae.png)

↑ http://kanboard.net [10]

I hope all these user interfaces will inspire you! Let's design the best open-source kanban board!

**Comment by ocdtrekkie** [comment]
Have you considered asking any of the people who have made their own redesign mockups directly if they would design for LibreBoard?

One thing I see in a lot of these that is a particular peeve of mine of the UI design some companies are using these days: Please avoid differentiating elements solely by varying shades of light grey. It can be difficult to see on poorly contrasted monitors.

**Comment by rusintez** [comment]
:+1: to Stu's concept. A great clean theme. Capacitive and forgiving.

**Comment by ocdtrekkie** [comment]
Another crazy thought: Could LibreBoard support multiple styles? There is a good point that it should not by default use Trello's. (Though it is common for users to host sites designed to look like other sites, and it may be acceptable to have a similar option.)

Then new styles could be tested or tried, and that could end up being a core differentiating feature as well. As traditional closed source cloud service products like Trello offer limited customization in that regard.

**Comment by crawford95** [comment]
While I like Stu's concept, it does not show nearly as much information on first glance as the current project or other mockups.

**Comment by mquandalle** [comment]
Inviting @stugreenham to the conversation;

**Comment by vohof** [comment]
Agree, :+1: Stu Greenham's.

**Comment by zakbach** [comment]
I for one would move on with the redesign as priority #1, right now it's a 1:1 rip-off of Trello and I'm not sure how legal that even is. As for the examples brought up above, regardless of which direction you'll take, I hope it's not a 1:1 rip-off of that either. Plenty of designers in the open source space, if you guys had a IRC channel or something, would love to help out.

**Comment by stugreenham** [comment]
Hi Guys,

Thanks for including me in the list. My reason for doing the redesign in the first place was because I was unhappy with the existing Trello look and feel. It feels dated, something I contacted Trello about but got nowhere.

Whilst I don't disagree with Crawford95's comment, I must add that this was done intentionally. I wanted to clean up the UI and remove anything that wasn't super important so that the focus was more on the lists and the cards. Simplify.

A user can still get to his/hers other boards, or invite another user in, etc but some of these features sit behind a click, something I felt I was okay with. As a user of Trello myself, I find that 90% of the time, I am just dealing with the cards.

Cheers,
Stu

**Comment by stugreenham** [comment]
May I add - I am more then happy to help out with this project if you need any help from a visual/ux point of view. Just let me know!

Cheers,
Stu

**Comment by ocdtrekkie** [comment]
@zakbach I'd move further discussion of copyright or trademark issues to #92.

@stugreenham There's some things about your design that's appealing, and I guess I buy the notion of hiding less used things like user controls in a menu. However it looks like you also hid the feed, which I consider pretty critical of Trello. In a large board, that feed is how you know if anything is different since last time you opened it.

My other note is that you seem to have nearly doubled the vertical space between cards. And the top bar seems to be very thick as well. I know companies like Google have been pushing incredibly bloated padding and margins as the "new" look and feel, but it feels like a huge compromise in space utilization for meager to non-existent benefits.

**Comment by Squ34k3rZ** [comment]
UX nerd here: Can you guys redesign the "Boards" button? :smiley: From an info architecture point of view, a side bar ("secondary nav") seems more appropriate. I can make a sketch later. Think of slack's ui where the rooms/convos are on the left in a column. Or your default notes application for mac. I think that's how a number of boards should be managed.

**Comment by stugreenham** [comment]
@ocdtrekkie I don't disagree with your comments - but I must stress that this mockup isn't to be taken too literal. I did this one mockup on a Sunday afternoon so I could play with Sketch. Had this project been a more then me just 'playing with ideas', then I would have gone into more detail and ensured all angles were covered and other screens and flows were sketched and designed.

I agree about the feed - but I must mention that this critical feed you speak of is also hidden by default on Trello. I hadn't ignored this or any of the other features and like I said above - this isn't a design that I was commissioned for. But I do think this is a good point.

Re the spacing - this is something I considered and from my experience dealing with Trello boards involving 15+ people I personally prefer the slightly larger spacing because I find the content more readable. When you add user icons/labels/comments and more to every card it soon becomes a chore to follow with your eyes. I had considered a compact mode similar to Gmail.

If you compare the header on Trello to the header on mine you will notice maybe 5 more pixels in height. Nothing hugely worrying there.

I'm happy to look into developing these concepts further if anyone is interested - This was purely a 'first crack' at it which tbh I am pretty happy with at a starting point.

**Comment by rusintez** [comment]
@stugreenham :+1:

**Comment by JamborJan** [comment]
I'm not a designer thats why I posted my proposal not here. But I would suggest to focus also on tags, how they can be organized and nested. For me tags work best when I have a huge pile of information pieces in front of me and I need to organize it.

https://github.com/libreboard/libreboard/issues/98

**Comment by ocdtrekkie** [comment]
@stugreenham I am a big fan of compact display density as a setting. :) I would still likely be a happy Google customer if they were willing to keep offering it on their newer products.

If the feed's hidden by default on Trello, I might just never have noticed. The sidebar does collapse, I suppose, though it remembers it's position, perhaps, because I never see it collapsed.

**Comment by Squ34k3rZ** [comment]
Everyone's talking about Stu's so I'll chime in:

Really good job, Stu

I think 'boards' button being on the top right might not follow convention. Normally it's primary links horizontally on top, secondary links vertically on the left side, and utilities or tertiary links on the top right.

I like the skeumorphic style. When distinguishing between this and the default ui, I'd say Trello has a more visceral feel of grouping cards because they're separated and contained within each dark area. Right now it just looks like a list and it isn't clear that they're cards in a container. I don't think people will immediately get this concept (that it's cards in a group) when looking at the software for the first time.

**Comment by ozodrukh** [comment]
Oops i wrote in another thread, about my design concept thoughts, i will repeat my message:

i'm not ui/ux designer, but i draw simple sketch of board in material like style, please criticize it we need do something comfortable to use
![screen shot 2015-01-23 at 12 16 22 am](https://cloud.githubusercontent.com/assets/3184950/5862733/ee3e1432-a295-11e4-92ed-0d001caac09f.png)

**Comment by ocdtrekkie** [comment]
Copying my notes here from #97 

@ozodrukh I will cry many tears if this goes material design. Many. Tears. #materialnono

I don't think a free software project should be implementing Google's branding. And material design has poor contrast between various layers.

**Comment by ozodrukh** [comment]
i like simple ideas in material design, yeah you are right contrast in material design is poor, but we can improve it here

**Comment by rusintez** [comment]
Hi everyone, just want to align the train of though here and please correct me if I'm wrong: 
— stu said that he might be able to develop his vision of trello ui further if anyone is interested. From discussion here it looks like there are 2 main directions to go:
- implement new distinctive look
- figure out new distinctive features / layout
While first seems to be a focused task the scope of which is relative easy to narrow down, the second seems to be a never ending improvement of ui/ux that not only touches how application looks but also how it functions (eg implementation might require backend changes, even db migrations). 

Since scope of first one is easier to pick on and easier to evaluate short term this is where I would propose to concentrate on. Eg in respect to his time, to ask Stu to further develop his vision of ~~trello~~ libreboard ui. Maybe concentrate on figuring out general ui guidelines - card, activity, settings etc. making it possible in the future for developers to extend ui using these guides in a way that product would evolve by that time. Again if you guys feel like it's the way to go.

**Comment by ocdtrekkie** [comment]
I think Stu's probably the best bet, if he's interested in taking it on, as I assume it's a fairly hefty workload. I'd really love if the ability to change out styles was actually a feature for LibreBoard in the future, as it'd be a particularly key differentiation from traditional cloud services. (Under such a system, something like Stu's could even be default, but it'd be possible to have a more Trello-ish style available as well, for example.)

**Comment by ozodrukh** [comment]
Stu's concept isn't great, i can't determine where is what, usability of his concept is poor, i think if cards will be implemented there, it really would be best bet

**Comment by ocdtrekkie** [comment]
@ozodrukh They're already cards... Everything on kanban boards is cards...

**Comment by rusintez** [comment]
Are there any metrics that could help us to stay on the path? Eg. What are the things that could be improved (in Existing design)? I think as users we could quantify usability issues (from our perspective) and leave it up for professionals to figure out the best way to solve those problems. There is no need to micromanage or introduce bias options like material design etc. 

I personally only figured 1 metric: 
- new layout should maintain a distinctive look to make product recognizable. Define that line where trello concept and libreboard concept floated apart. Criteria of success would be a layout that inherits all existing functionality (no judgement of what goes or stays at this stage) plus it's flexible enough to take in some realignment at the time new features land on to the project. 

I believe guys started this thread in the first place to find their own identity whilst having a default trello feature set bro start with. 

Any ideas ?

**Comment by neandrake** [comment]
I have a few comments regarding small improvements to the current interface. With the UI redesign plans in this Issue, I wasn't sure whether improvements to the current design would be continuing. I can make separate issues for these, but I figure I would just mention here first.

1. Popup menus should close when their button is clicked again. Examples of this include the Profile/Name menu in top right, the New Board '+' button in top right, a card's options button in the card's top right, etc. Currently you have to click somewhere else on the page to close these popups.
2. On the flip-side, the click-to-edit paradigm currently requires clicking on an 'X' to cancel/hide the editable text field. This form should cancel (or save?) when clicking elsewhere on the page and lose focus. Example of this include clicking a list's title to rename. I don't think this should apply to forms with multi-line textareas as this generally indicates a large amount of data can go in, and this should not be lost when changing focus - example of this would be adding a new card form.

**Comment by aleksandar-todorovic** [comment]
I really hope that Stu's solution will become the default look of the project!

**Comment by frdmn** [comment]
+1 for Stu (but with the feed)

**Comment by bmathews** [comment]
Yeah, @stugreenham's design is great. Stu's a real UI designer, how about we let him call the shots on aesthetics.

**Comment by BenjaminCorey** [comment]
I think that all these concepts are great, and while I agree that a nice theme and out of the box installation is great... I think it would be good to take a queue from the bootstrap team and abstract the styles from the core project. This allows everything to remain lightweight and customizable.

Obviously, this takes much more planning than just choosing one style, but it allows for much broader adoption if users aren't forced to reset a bunch of gradients, drop-shadows and sprites when reskinning.

**Comment by cjb** [comment]
There's also the design used by https://taiga.io -- I think they have a great design, and their code's open source already:

![screenshot from 2015-01-23 20 28 08](https://cloud.githubusercontent.com/assets/21217/5885577/ab9e0a8e-a33e-11e4-9f86-7990ccf572ab.png)

**Comment by ocdtrekkie** [comment]
@cjb Soooooo much wasted space o_o

**Comment by Squ34k3rZ** [comment]
As for the number of lists visible on the screen (3? 4? 5?), you could make the argument for 4. We used to think we could hold 7 things in our short term memory, but recent research has shown it's more like 4 or 5. I actually like trello's sized lists on my laptop screen because there's 4 and a half. The half list indicates that you can scroll right to see more lists. I think that's the perfect number of lists to be on a screen at any one time.

**Comment by ocdtrekkie** [comment]
The number of visible lists depends on your screen size. Trello fits six and a half on mine. (Maybe you need a better screen?) Large high resolution screens rightfully should display more information than smaller screens.

**Comment by Squ34k3rZ** [comment]
@ocdtrekkie True, you make a good point. For laptops, I guess I'd like at least 4 dude. And can we make this responsive for mobile to have just one? Here's a sketch. Sorry it looks really polished. Trust me, it took just a few minutes to make.. So feel free to criticize the heck out of it! I'm going to make a sketch of a secondary nav I'd like to see that can distinguish our app from trello (and avoid DMCA complaints)
![screen-shot-2015-01-23-at-10 42 49-pm](https://cloud.githubusercontent.com/assets/4743518/5886192/d9ee7170-a353-11e4-8919-3468e7db0ba6.png)

**Comment by ocdtrekkie** [comment]
It would make sense for mobile to have just one wide before scrolling.

As for your mockup, I'd probably suggest the top bar should be more distinctly different than Trello's in style as well.

**Comment by Squ34k3rZ** [comment]
Here's the sketch for some ux improvements. Basically everything you need to be productive. The notes is placed at the bottom, so your list of notes can grow into infinity and all you need to do is scroll down. (Although a load more button is possible).

You could put a search bar up top as well.

Think Slack + Trello + Notes ftw
![screen-shot-2015-01-23-at-10 42 49-pm](https://cloud.githubusercontent.com/assets/4743518/5886219/c7f7da36-a355-11e4-9519-99a7b524e0ca.png)

**Comment by gmaggio** [comment]
Definitely Stu!... at least in terms of the overall feel and direction...

**Comment by tj** [comment]
This one gets my vote:

![](https://cloud.githubusercontent.com/assets/1730702/5819605/61a751be-a0bf-11e4-8651-68368efbe1b8.png)

Trello is soooo ugly

**Comment by vegetableman** [comment]
+1 for the ui by Duncan Malashock

**Comment by litera** [comment]
Definitely @stugreenham design. Stu: add parts that others have requested and maybe upgrade it with your latest ideas and provide another UI mockup for a review. This should move a bit faster.

**Comment by ryngonzalez** [comment]
Hi all, interface designer from Fetchnotes here!

I think one thing that's missing from this conversation is a high level set of goals/usecases/niceties that we want to serve with this design. Before we start trying to find an alternative interface, we should be figuring out the shape of the problems it'll try to solve. 

If we're trying to copy the goals and usecases of Trello 1-to-1, then we're inevitably going to trend very closely to their product design, with only the room to make superficial changes (re-jiggerred navigation, changed visual style, etc.) but I think taking some time to figure out our fundamental differences and perspective on the problem space (a spatial, manipulable layout of information that you can use with other people) will be very useful to how this discussion moves forward, and how us designers can help contribute.

**Comment by mquandalle** [comment]
We released v0.8 yesterday (with a lot of new great features in it!), and it's now the time to fully focus on our new UI. Due to the DCMA notice, we won't push any code on github, but we'll keep working in the open on a self-hosted [gitlab instance](http://git.libreboard.com/libreboard/libreboard). I've also set a separate instance of the online application on http://newui.librebroad.com so we can share our experiments and break things.

I'm not certain @yasaricli agree with me on that (please share what you think!), but for my part I don't want this re-design to be “just change the CSS so we can get back on github”. I'm willing to work on much more deeper modifications to the way the application work (how we assign members and label to a card in the board view, how the multi-selection feature will work, how we select a board, how we integrate pomodoro timers, etc.). So if you're thinking about how we can improve things, don't feel limited by the amount of code re-write you're idea will require.

There is also the mobile. We're not going to develop native LibreBoard applications on Android and iOS, instead we will wrap the web client in Cordova (which is easy to do with meteor by the way). I don't want to abuse marketing words, but “mobile first” could be a good approach for our new user interface.

@ocdtrekkie Multi-design would be a very cool feature indeed. Unfortunately, due to the way meteor works (take all the CSS files it can find and merge them in one big file) this is not something we can implement easily. So let's keep this idea for a later release.

@ryngonzalez I would love to tell you that we have a vision about the way we want to spatially organize information. But as far as I know our only fundamental difference is to be a free and open-source software; I'm not sure we can emphasize data privacy in the UI (I don't think so), but maybe one point to consider here would be plugin extensibility.
Also, if someone has some ideas to improve the kanban paradigm, feel free to share them (@JamborJan proposed #98). Maybe the multi-selection idea could have some ramifications? For instance we could make the labels click-able to open a menu like “Remove this label” + “Filter cards on this label”, and have a similar menu on the member icon.

It seems that there is a consensus to use the mockup of @stugreenham as a basis, and iterate on that (side note commenters, keep in mind Stu didn't originally made this mockup for our use case). Stu, could you authorize others to iterate on your design? For me, this mockup raises a number of issues we need to figure out:

* Do we need a sidebar (similar to the Trello sidebar)?. I tend to think it would be possible to move all board actions to the top board bar. Maybe this top board content could also change depending on the context, for instance become a “drop here to archive” when we are dragging a card (like on the android version of Trello), or became a filter selection if we are in the filtered view. Non-frequent actions could be moved to a menu accessible by cliquing on the gears icon on the right. However we definitely need a sidebar on the mobile UI.
* How do we keep a level of board user customization? Should the user be able to select a header color? Do we want to support background images for the header (a bit like chrome themes) while we keep the content on a white background?
* How do we select a board? It seems that the top left hamburger icon could open a left pane with a board list?
* How about white cards on a grey background, like Google Now?

There are also potential discussions about square vs round avatars, or about the way we display labels on the board view (anyone like the way Fabrice Liut [1] solves this?), but I don't have a strong opinion on these.

And we also need a new logo #64.

**Comment by cjb** [comment]
> we'll keep working in the open on a self-hosted gitlab instance

Wouldn't you just expect to receive a DMCA notice to the gitlab instance, then?

> for my part I don't want this re-design to be “just change the CSS so we can get back on github”.

I'm nobody here, but I'd suggest the opposite approach, so that you have a project that people can use, develop and deploy in the open without waiting for your full feature design and implementation to finish first.

**Comment by yasaricli** [comment]
@mquandalle I agree with you.

**Comment by Squ34k3rZ** [comment]
"Do we need a sidebar?"

I haven't used the activity sidebar feature, but then again, I do most my work on jira at my real job. That's an interesting design question worth asking everybody here. How much do you love the side bar on trello?

Also, we can group some of these functions together and put them in a Settings modal to keep the interface clean. 

I like the hamburger icon + list instead of selecting from a boards page

The gray sounds good. One thing to consider is the visual weight of your top navigation bar. The darker gray you go, the less visual weight the dark navigation bar will have. So while it makes the white text boxes pop more, it makes the whole page pop less. That's why I used light grays in my design (the red one above). Your call, boss! Just providing you with information to make your decision.

**Comment by Squ34k3rZ** [comment]
@ryngonzalez makes an excellent point! One thing I noticed using hipchat + jira (it's like slack + trello) is back and forth between chat window and project management board. It'd be cool if we could use sockets.io to let people select things, like you can have multiple people edit the same document in Google Docs. I'd love to see a box with a highlight around it from my project manager. It'd be even better if we added chat and kept it to the same website. I showed this possibility in the red sketch above where the side navigation has Private Messages, Chat Room Channels, Trello Boards, and a section for Notes. That's a holistic approach to project usecase, if you ask me.

**Comment by mquandalle** [comment]
Also in most mockups the logo is tacking a significant area. I don't think we care enough about our brand to take all that place, and I believe we don't even need to put the logo in the board interface (just keep the application name in the window title).

**Comment by dannydewit** [comment]
+1 for Duncan Malashock

**Comment by alireza-ahmadi** [comment]
+1 for white card instead of grey

**Comment by guarilha** [comment]
@stugreenham :+1:

**Comment by pertrai1** [comment]
I say get the CSS changed and get it back up here so all of us can help make the changes that are needed for this project. Take out what Trello is considering to be theirs and then there are no issues.

**Comment by andreioprisan** [comment]
This doesn't have to be a complete UI paradigm shift to get back on GitHub. There is only so much that you can change while still keeping the cleanliness and simple flow that is inspired by Trello flows.
Just remove offending hot-linked assets and styling that is 100% original to Trello so that others can continue to contribute. 
@mquandalle @yasaricli I can take a stab at a UI cleanup to get this back here so that other people can continue to contribute. Short of forking this and taking it a different direction, how could I contribute?

**Comment by mquandalle** [comment]
@andreioprisan if you want to help you can send a merge request on http://git.libreboard.com/libreboard/libreboard

**Comment by mquandalle** [comment]
I started working on a new design today. What I have below is an actual implementation (not an image). A lot of features are bugged due to some code reorganisation, so I'll try to fix that before I release the code (hopefully tomorrow, or after tomorrow).

![capture d ecran 2015-01-25 a 21 46 21](https://cloud.githubusercontent.com/assets/1730702/5893225/d1ba620a-a4db-11e4-997c-55e5f045a741.png)

One of the idea here, is to have quick access to starred boards on the small top bar, this is a feature inspired by reddit sub-reddit. I also removed the sidebar and put all the board information in the top bar. This bar will change depending on the context, for instance if I'm on a filtered view the bar will display the filters, if I'm dragging a card the bar will display a “drop here to archive” button, and on the home page, the bar will just display a LibreBoard logo.

Also I moved the menu that was on the sidebar into a popup that is accessible by clicking on the board title. For instance to change the board color:

![capture d ecran 2015-01-25 a 21 51 49](https://cloud.githubusercontent.com/assets/1730702/5893243/62fef924-a4dc-11e4-9347-8ce2ea78f758.png)

As I said, this is a very early prototype (not even 24 hours!), so I haven't worked on typography, space rhythm, clean animations, color adjustment, etc. I just like to share what I have done today :-)

**Comment by orges** [comment]
That is clean work! Are you planning to put a historical activity log pane?

**Comment by mquandalle** [comment]
So my ideas around activities:

* On the home page have a feed of activities for all starred board, and all followed members (like github)
* On the board page click on the board menu then “Activities” and that will open an overlay window containing all the activities for this board.
* We also need to redesign activities UI, personally I love how github/githab draw their feed with a vertical time line with messages and events. Since the gitlab design is released under the MIT license I might just use that for our activities feeds

**Comment by gomex** [comment]
@mquandalle great job! If you need help with gitlab infrastructure please let me know :)

**Comment by Coppolaemilio** [comment]
@mquandalle Looking forward to start contributing :)

**Comment by mquandalle** [comment]
@Coppolaemilio Cool! What kind of this would you like to do? Implement new features? Help with the design?

**Comment by fnky** [comment]
Took a shot at this — very unfinished, and everything is subject to change.

![Libreboard](https://cloud.githubusercontent.com/assets/995050/5893823/2785500c-a4f3-11e4-85d1-641b1e90bd13.png)

**Comment by Telemakhos** [comment]
Love @fnky proposal. Really clean and focused on the cards... :+1:

**Comment by Shekar73** [comment]
@mquandalle early look of the design is looking much better. Actually much much better than Trello :smile:, which is now looking like meh.. compared to this.....

I know this is very early design, but the dotted lines  between the lists are little distracting...

**Comment by Squ34k3rZ** [comment]
love what @fnky did

Idea: just brainstorming here, if you moved the green button to the left side of the box, would that make it more scannable? I like how you have it as well.

I think it's looking really good in terms of actual interface changes. What would the lists look like if they were even like a darker gray or a brownish dark grey with white text instead? Just curious.

Good work!
Tyler

**Comment by Squ34k3rZ** [comment]
@fnky Check this out. Tell me what you think

Just a quick sketch we can perfect later

![sketch](https://cloud.githubusercontent.com/assets/4743518/5894183/9b40651a-a4c4-11e4-9dc7-5bc6ae4a64dc.png)

**Comment by Squ34k3rZ** [comment]
Here we have buttons moved left for scannability

![sketch](https://cloud.githubusercontent.com/assets/4743518/5894206/4534b4a4-a4c5-11e4-887d-a1a5e0be95be.png)

**Comment by Squ34k3rZ** [comment]
@mquandalle Nice sketches as well! Going to be tough to decide which format to go with :) I can see where you're inspired by Google Now with the headings being outside the lists and as part of the background.

**Comment by fnky** [comment]
@Squ34k3rZ Thanks for the input :-)

I tried different ways of presenting boards/lists. I took the decision for a light background, that didn't feel too off the overall design and still preserving the functionality

- Can be moved
- Groups tasks/cards and visually shows its volume.

As for the buttons, I chose to put it in the right side, as this layout is used widely throughout every day applications—both on the web and OS').

They could also fill the whole board/list, but that feels too bulky.

**Comment by sumitarora** [comment]
@stugreenham :+1:

**Comment by denisinla** [comment]
@stugreenham nice and simple. Now to move on to implementing.

**Comment by stugreenham** [comment]
@mquandalle Just caught up with the thread - I'm happy to discuss the potential of using my mockup design but I would want my input. There is a lot of different opinions on here (which is good btw) and I'd be worried that once everyone has there input we could end up with a car crash of 100 different styles and ideas.

I think a lot of people are focusing too much on theming and colours etc. I've always thought of this feature on Trello along with their stickers etc to be an unnecessary distraction – If you are worrying about the colour of the board and not the content then you are doing something wrong.

What did you have in mind going forward then? Is this purely going to be web based for now?

**Comment by mquandalle** [comment]
Hey @stugreenham, happy to read you here again. Did you see my last two comments in the flow? https://github.com/libreboard/libreboard/issues/94#issuecomment-71327633 and https://github.com/libreboard/libreboard/issues/94#issuecomment-71393038 (←I started an actual implementation)

**Comment by Telemakhos** [comment]
Other interesting features for the new UI are collapsible cards and  [swimlanes](http://i1.wp.com/blog.david-jensen.com/wp-content/uploads/2013/04/Greenhopper-Multiple-Backlog-Rapid-Board.png), like in Jira Agile...   @mquandalle you should install UserVoice or something similar in your website so people can vote for the preferred features... or maybe you could open a board where people can vote...

**Comment by douglas** [comment]
@Telemakhos I agree 100%, uservoice would help so much here.

**Comment by leobalter** [comment]
@mquandalle++ I'm appreciated for all your efforts to bring a new UI to this awesome opensource project. 

As I didn't catch up everything in this thread, I would like to suggest you - if it isn't yet - to create a new libreboard project to create the new UI, without any functionality, maybe from the scratch, but accepting PRs and specific issues (as this one is already huge).

With that, we could even try different paths for the UI and have a progressive check for anything different from the main functionality or copyright issues.

**Comment by timotk** [comment]
The top bar with the tags from @mquandalle looks great and very clean.
The cards with small icons and sidebar from @fnky are also great. 

A combination of these two styles would get my vote.

**Comment by litera** [comment]
Ok. Let's first get a confirmation from the owners which design should be implemented. My opinion is (and I see majority thinks the same) that Stu's version should be implemented. A developer can only design an interface as good as they know, a designer (and possibly a UX designer) can go even further.

I suggest Stu's design be implemented and guided by Stu himself, so he can tell implementers what's invalidly implemented and which parts should be included as well. And he's right when he says that colour is not important. That's always right in design. If it works in BW it will work even better in colour. So UX first and then design as per Stu's guidance.

**Comment by fnky** [comment]
:+1: I agree with @litera — We need to organise tasks for anyone who would like to participate in putting this together as a team, not as individuals.

It's important that we don't waste our time discussing the look & feel of the app, but rather start discussing the functionality and the user experience.

**Comment by mquandalle** [comment]
Give me a few days to implement a design proposition and release the code; then we will be able to collectively iterate on that.

**Comment by mquandalle** [comment]
For the curious, here is where I am currently (again it's an implementation not a mockup):

![capture d ecran 2015-01-27 a 16 34 53](https://cloud.githubusercontent.com/assets/1730702/5921192/9077025e-a642-11e4-9212-6b2881677332.png)

There is even more bugged features than last time because I've been touching code everywhere. But nothing to worry here, I just need some time.

**Comment by Telemakhos** [comment]
Np, take your time pal. What we need is a way for proposing and voting the best features... maybe open a dedicated board in libreboard.com? With community feedback we'll know what tasks should take priority...

**Comment by bmathews** [comment]
Looking great @mquandalle. I think the work you're doing is a great combination of some of the proposals. 

I think you could ween some more simplicity/clarity out of it by sharpening up the shadow on the cards and ditching the embossed lane borders.

![image](https://cloud.githubusercontent.com/assets/848347/5923475/cbcbbed8-a608-11e4-93e3-b2694067391a.png)
http://jsfiddle.net/c850pa0m/1/

**Comment by Coppolaemilio** [comment]
+1 to @bmathews proposal. It looks much nicer with that clean style.

**Comment by mquandalle** [comment]
Thank you very much @bmathews, this is a great feedback; I've updated the stylesheet to follow your recommendations. You had to be motivated to create a complete jsfiddle! This is going to be great

**Comment by Shekar73** [comment]
@bmathews  :+1:  it certainly looks soooooo much nicer....it feels so much simple, clear and elegant!

**Comment by tj** [comment]
hahah wow I missed this part "We received a DMCA takedown notice from Github regarding our user interface", that's pretty low, not like Trello looks good... at all... or does anything good really

**Comment by mquandalle** [comment]
I'm still not sure about the “should we have a sidebar” question. My initial idea was to differentiate from trello and simplify the layout by merging the sidebar into the header bar. If you take a look at alternative kanban boards on my first post you'll see than none of them have a sidebar. 

On the other hand a retractable sidebar like [the one @Squ34k3rZ did](https://github.com/libreboard/libreboard/issues/94#issuecomment-71403454) seems useful, and difficult to improve. It gives informations (board members, labels, and activities) that other kanban software doesn't provide as easily.

Any thought on this topic?

**Comment by JamborJan** [comment]
From my point of view it would make sense for easy selecting tags to navigate through big amounts of cards (see: https://github.com/libreboard/libreboard/issues/98).

**Comment by ocdtrekkie** [comment]
The biggest thing I need from a sidebar or similar UI element is the
ability to see the recent history of people manipulating cards.

-Jake Weisz

**Comment by cpursley** [comment]
Perhaps have notifications always shown on large viewports. And auto collapse it on x size view ports?

Also, some great ideas here around Material Design: https://www.polymer-project.org/components/paper-elements/demo.html#core-toolbar

**Comment by Shekar73** [comment]
@mquandalle I vote for having the sidebar and @fnky and @Squ34k3rZ design fits well. If  I were to put  the users shoes  for a minute,  if I am working on a project,  I like my stream of activities and any changes made to the cards close by, so that the information can be accessed with no friction, rather than going to a overlay screen to access the information, which might lead to cognitive dissonance as I am breaking my train of thought, as the  app created an additional step to get to the information I need. The reason the sidebar works the way it works i guess, is very similar to how we all work, in the sense when we are working on a project we want to feel and see that the   necessary information for the project is close to us  and can easily get to it, with out taking any extra steps . Sometimes if something fits well in a absolute sense within the good usability   framework and solves a problem, then it is best to stick with it, regardless of who started it first.

**Comment by Shekar73** [comment]
@mquandalle  Actually from a color rhythm   point of view, I like @fnky design better. Having a main body with grey background and white cards, while having a activity sidebar with  white background and grey boxes for activity and event text, creates a simple, clear contrast and makes it more elegant from a UI perspective.

**Comment by ocdtrekkie** [comment]
I like the @mquandalle WIP better than @fnky mockup. Colors look cleaner, and less space is wasted on mquandalle's. You only need so many nested borders on a page!

But it needs a history sidebar.

**Comment by ibuilder** [comment]
You could just use twitter bootstrap:

http://bootsnipp.com/snippets/featured/kanban-board

**Comment by kiswa** [comment]
Don't know if it's 'modern' enough, but I thought the look of [TaskBoard](http://taskboard.matthewross.me) might at least give someone some ideas.

![TaskBoard](http://taskboard.matthewross.me/images/board.png)

**Comment by catmanjan** [comment]
Is there an ETA on when "we'll be back"?

**Comment by ocdtrekkie** [comment]
@catmanjan Given that the preview @mquandalle gave above was like two days of work, I imagine it will not be very long.

**Comment by Shekar73** [comment]
I agree with @ocdtrekkie . @mquandalle  is a coding badass :smile:

**Comment by Squ34k3rZ** [comment]
@mquandalle I think one thing we need to consider is our user. Is he going to be working alone or on a team?

If on a team, I'd recommend a side-bar with the extra functionality (such as chat, 1 on 1 messages, and notes)

If he's just doing his own thing, then those might be superfluous features.

The other nice-to-have feature to roadmap is Natural Language Processing and identifying the cards. And then when people chat, being able to refer to each card with a #searchterm

Depends on our resources/ability to implement obv. I'm just going by actual behavior out in the wild. Another thing we use is gchat to voice/video chat. So if we could add a group voice feature to the chat rooms, that would be mega. ;) Plenty of possibilities to work on!

**Comment by fnky** [comment]
Took some of the ideas into consideration and made a few changes + an extra mockup without sidebar as [@mquandalle](https://github.com/libreboard/libreboard/issues/94#issuecomment-71844125) suggested.

![desktop hd](https://cloud.githubusercontent.com/assets/995050/5956662/6f997962-a7b0-11e4-842f-43ce1bde8412.png)

![desktop hd 2](https://cloud.githubusercontent.com/assets/995050/5956667/74b856b6-a7b0-11e4-8e10-c22299b88f49.png)

---

@Squ34k3rZ I don't quite see the idea in implementing a chat system in a Kanban board, since it can easily be integrated with a wide range of services (like Slack, HipChat, etc...)

It's important to make one thing and that thing well. Implementing chat, voice- and video is certainly something that takes time to implement, which isn't mandatory as there is lots of services out there.

As for the audience, we will never know, but Kanban is a widely used method for small to large groups of people and it'll probably be picked up by any kind of company. Therefore it's important to discuss about features that are useful to Kanban.

**Comment by alvinsight** [comment]
Are you guys considering having the panel/menu on the left instead ? I just think it might prevent some additional legal issues

**Comment by andreioprisan** [comment]
@alvinsight implementing or left or right is not a legal issue in and of itself. The issue was with the 100% style copy, which we can't do. Having the same data feed there would be OK.

**Comment by andreioprisan** [comment]
@fnky I love the view without the explicit right sidebar and being able to just toggle the Activity feed if needed.

**Comment by mquandalle** [comment]
Thank you @fnky, your mockups are inspiring. I decided to give a try to the no board sidebar approach. We could then use a sidebar on large screen to display card detailed view (you can discuss this idea in #127) and for the board activity feed — but not for labels and members that will be in the header.

Also we don't need a filter selector like the one you drawn, we will be able to filter cards by clicking on a member avatar and selecting “Filter cards with this member”. The top bar will then change to display a big "X Filter view" title and displaying all the active filters. Clicking on "X" will reset the filter. I will implement this in the coming days so you can share what you think of it.

Here is the current state of my implementation:

![capture d ecran 2015-01-29 a 19 17 34](https://cloud.githubusercontent.com/assets/1730702/5963339/825a59dc-a7eb-11e4-94b2-2c05182d0834.png)

I imagine a lot of graphical details could be discussed from it. Round avatars? Border-less covers? Improving "Add a card/list" buttons? Also I don't really like the Starred/Labels/Search top toolbar; I will probably try with bigger icons and no text, a bit like in the @fnky mockup.

Any feedback?

**Comment by mquandalle** [comment]
Also since there is already more than 100 messages here, I propose to split the new design discussion in multiple thematics threads:

Re-design discussions:
* #126 For the home page and the activity feed
* #127 For the board general view
* (current) For the card detailed view

The landing page will also be redesigned, but later.

**Comment by mquandalle** [comment]
@Squ34k3rZ LibreBoard will be used by both individuals users, and teams (since it synchronize the view in real time) — like Trello. And LibreBoard is not only available in English but also in French, German, Japanese, Chinese, Turkish, and Portuguese (more are coming).

**Comment by tjfwalker** [comment]
I think it'd be great if users were able to pick/swap which "skin"/UI their LibreBoard had. It could be an option on settings or something.

**Comment by Shekar73** [comment]
@mquandalle Wondering from usability point of view, the header might get crowded if there many members (for example 10-15 members on one board). My opinion is, I like the current implementation state of the board but with sidebar component  as designed by @fnky (or maybe the sliding sidebar  might be a middle ground?)

**Comment by litera** [comment]
@Shekar73 Not all members need to be displayed at the same time. There can only be a handful 7+-2 and a dropdown. A similar approach is done on Google Docs where one can collaborate with users that are on the same doc at the same time.

Go check [this Google Doc](https://docs.google.com/document/d/11YUzC-1d0V1-Q3V0fQ7KSit97HnZoKVygDxpWzEYW0U) that usually has many many concurrent viewers. Cklick link and wait for the document to completely load and check the upper right corner user list.

Users are anonymized as per Google's implementation, but that's not the issue here.

**Comment by fnky** [comment]
Thanks for the response @mquandalle 

When I initially made the mockup, I thought of adding labels with explanations of the components. I like the approach that having a sidebar by demand on larger screens, and keeping everything tight but still available on smaller screens.

The **Filter** dropdown should actually be a **Labels/Tags** dropdown. I already had in mind, that you filter by members by clicking on the members avatars, since you'd be able to also filter by more than one member. However, only about 7 avatars will be visible, so that'll need a dropdown as well to list all members within a project and filter by them.

Filtering by tags is just a matter of opening the tags dropdown and select the tags you'd like to filter by and searching would be a smart search with commands and keywords.

We'd also need to be able to filter by tasks assigned to you, and perhaps also unseen activities.

I like the approach that when you're filtering, it'll show you the filter settings. This gives the user a notice that the data they see is filtered and they can change the filters or reset them.

**Comment by pwmvx1289** [comment]
Should the menu has transition animation when being hide/shown?
This way the actions would seem more natural and fluent.

**Comment by xasos** [comment]
Stu's or Fran's are my picks

**Comment by JamBoi** [comment]
Right now there seem to be serious issues with logging on to Libreboard.com . like you go to the login screen, put in your info, hit submit and nothing.

**Comment by conmer** [comment]
Might not be helping much but I would love to see this plugin's UI on Libreboard.

https://github.com/MikoMagni/Trello-Monochrome-Chrome

**Comment by Squ34k3rZ** [comment]
Playing around with background styles....

![screen shot 2015-02-03 at 11 23 14 am](https://cloud.githubusercontent.com/assets/4743518/6025336/427f2f18-ab97-11e4-90c4-9ad00dee3645.png)

**Comment by orangejuic** [comment]
Let's try to establish some rules.
1. the used fonts MUST not be too extravagant like in @Squ34k3rZ 's card titles right above me or in taiga.io 's extravagant design; it makes them harder to read and that is absolutely something to avoid, I think we all agree there.
2. I see a lot of people are liking Stu's design, however I believe having color is very important to have an easy overview of either what the cards contain (color in function of content) or freedom to choose the color of a card upon its creation (personally I'd be more inclined to the former, which would prevent a lot of headaches I believe)

What do you think?

**Comment by litera** [comment]
@orangejuic regarding #2. It's not that anybody opposes usage of colours. I don't think that majority like Stu's design because of that but rather of the *execution*, how well vertical space is used by adjoining cards visually separating them by line only. This gives cards more breathing space and doesn't confine individual ones inside strictly defined borders. That's the great thing about his proposed redesign.

This way each "list" becomes a column of visually joined cards. Why people relate to that so well? Because it better mimics real world. Imagine a newspaper where each individual article would be put inside a visually defined container. Articles in newspapers run in columns and usually not being separated by any other marker than whitespace and their content emphasised parts (titles separate individual articles in the same column vertically, whitespace separates columns horizontally)

So that's the whole beauty of Stu's design. I wouldn't use black background either. Actually I'd try doing it even more like real world and keep background white and individual lists without any borders. But in this case majority of card data should be left aligned to relate stuff correctly (i.e. avatars belong to the card with data on the left, not to ones on the right)

Something along these lines:

![image](https://cloud.githubusercontent.com/assets/800583/6168358/fe6832b8-b2c4-11e4-98c2-b4b19addb804.png)

**Comment by ocdtrekkie** [comment]
@litera The UI in the one you just posted is so faint, it just appears to be floating blocks of text on my screen.

**Comment by litera** [comment]
@ocdtrekkie I'm not sure what you mean or what kind of a screen you're using, but the main idea to post the UI was to show that Stu's design isn't so much about colour as it is about individual elements design. I've maybe done it all to light. It displays fine on my monitor though.
My changes are just put directly on Stu's original design. It should of course be greatly improved upon as I was just removing or changing a few bits and pieces.

**Comment by orangejuic** [comment]
@litera I wasn't saying stu's design is entirely bad, just that the lack of color is a flaw that has to be remarked upon. Now about that design you posted, I believe @ocdtrekkie to be right in that there are no cards anymore, only floating blocks of text, simply because the separation between the cards and the columns aren't clear.
From afar, I can't distinguish clearly the cards in your design, and that's a major flaw in my opinion because a card is so much more than only text. It's supposed to be designed to contain files, images, recordings, a video or even a link.
The design you posted doesn't seem to be adapted to all of that, because it's made to contain a bit of text and that's it, when Trello (the base idea of this project) is made to contain so much more.

**Comment by erlend-sh** [comment]
What started out as useful brainstorming has devolved into design-by-committee bikeshed discussions.

@mquandalle, @orges  and @yasaricli , if you're gonna make a bet on @stugreenham's design - which it looks like you are, judging by your most recent mockups - it's an all or nothing deal. The community has done its part for now. Stu's design was the majority's favorite. You don't need 10 contributors to accomplish what comes next, no matter how cool and intoxicating it is that so many people have been willing to pour so much energy into this. (I've hardly seen anything like it, *so cool!*. I'm sure it'll come in handy again when it's time to open the floor for smaller iterations and possibly alternative themes if that's ever supported).

In Stu's own words

> I'm happy to discuss the potential of using my mockup design but I would want my input. There is a lot of different opinions on here (which is good btw) and I'd be worried that once everyone has there input we could end up with a car crash of 100 different styles and ideas.

This is exactly where you are headed right now. There's no room in here for a single, unified direction.

My 2 cents: Stop this discussion, get in touch with Stu and finish the design that you already know is fully capable of wow-ing people.

**Comment by jasondavis** [comment]
so many great designs!

**Comment by litera** [comment]
@erlend-sh Exactly my sentiment. I've pointed somewhat to this same issue some [time ago - 27th Jan](https://github.com/libreboard/libreboard/issues/94#issuecomment-71627700) but nobody seemed to reply upon it.

**Comment by JayBeavers** [comment]
@mquandalle Just wanted to pitch in that http://newui.libreboard.com is looking great.  I agree with @erlend-sh that the value now is in getting that delivered in a 0.9 release that allows you to come back to Github.

Looking forward to 0.9 so I can move my team over to libreboard!

**Comment by stephenjlewis** [comment]
@mquandalle, I do like the visibly clickable tags on @fnky's mockup: it's intuitive, much moreso than a nested menu might be. If we're avoiding a side bar (are we?) there may be some way to integrate the coloured tags into the top bar for easy topic-level filtering.

**Comment by pdaoust** [comment]
Pardon me if I'm missing something, but http://newui.libreboard.com/ looks nearly identical to Trello and doesn't seem to incorporate any of the new design directions discussed here. Is there a new URL for the new UI?

**Comment by leecade** [comment]
:+1:

**Comment by fix** [comment]
I have just redesigned in the code source the app. If you are interested, i'll push it on the source.
![screen shot 2015-03-24 at 20 58 05](https://cloud.githubusercontent.com/assets/709388/6811532/db17bacc-d268-11e4-9db3-54a7c1115ce4.png)


![screen shot 2015-03-24 at 21 02 50](https://cloud.githubusercontent.com/assets/709388/6811591/37e6165e-d269-11e4-9520-95d9bc88799a.png)

**Comment by Shekar73** [comment]
@fix  :+1: :+1:  Love  the custom board backgrounds and chalk board metaphor in the home page!!

**Comment by ojosdegris** [comment]
@fix this is really cool and useful for branding!

**Comment by yrshaikh** [comment]
So is there a downloadable version available ?

**Comment by SimonSarazin** [comment]
Some friends worked a year ago on a kanban design. Maybe you would like to use this design. I have to confirm, but i think it's opensource. You can see and test this it : http://kanban.apps.patapouf.org/ login : demo et pwd : demo . I think the app will never be updated as a bigger community exists to develop this common, we prefer to support libreboard. I'll ask mockup to the designer (and psd files) if that's in your interest

**Comment by andyburnett** [comment]
Wow, I think that design looks great!  I would vote for using it.

Cheers
Andy

**Comment by Angelg1409** [comment]
excellent design !!
+1 for me

Angel


2015-05-12 15:59 GMT+02:00 SimonSarazin <notifications@github.com>:

**Comment by SimonSarazin** [comment]
Here are the complete designs : https://drive.google.com/folderview?id=0B1GiMK8ptm02fjRYazFXTFdSaWRBcUgyNU1BZU5oMUpVRDRUMExENVMtZEw1d292TWZJWW8&usp=sharing . It is in creative commons (just please keep the designer paternity if libreboard finally use it). All is in one psd that i can share if you need this design.
![17](https://cloud.githubusercontent.com/assets/620174/7656199/61803254-fb29-11e4-9991-575c64f1c927.png)

**Comment by omeid** [comment]
:+1: For Stu's Design. So how is this going? have you guys decided yet to go towards any specific design? I could help here and there.

**Comment by omeid** [comment]
https://userstyles.org/styles/browse/trello

**Comment by TheElf** [comment]
I am fond of Stu's but would like some more material design elements added because I think they would make the UI/UX language more clear

I agree with @Squ34k3rZ about the slide out left menu, I think that floating menus can often cause problems visually. 

There are actually several designs on here that use slide out menus that float above other text and I think it would be really helpful to implement material design concepts on top of this because it makes the separation a bit more obvious. 
Near the beginning of the conversation, someone mentioned multiple shades of gray as being problematic on low quality displays. I agree with this, and also think that the slight shadowing between levels in material design alleviate this. 

just my 2 cents.

**Comment by ocdtrekkie** [comment]
@TheElf Material design, is, by definition, terribly unclear language. (There's a recent article by Lauren Weinstein, who has actually done a lot of work for Google, about how 'modern' UIs tend to fail badly: http://lauren.vortex.com/archive/001112.html)

**Comment by TheElf** [comment]
Very interesting read Jacob, thanks for posting that.
I myself have found it to be very helpful, but I don't suffer from the
problems listed in the article. (I'm sure they will be problematic for me
in the future, just not now)
The things that I was highlighting material design for when recommending it
in my post were more in regards to visual separation of elements with depth
(cued by shadow). This isn't something that the article took issue with,
and is still not something that existed in many of the posted mock ups.

Now this may have been because they are simply prototypes, but I think it
makes a big difference in how readily identifiable different pieces of a
page are.

**Comment by ocdtrekkie** [comment]
@TheElf I suppose it's definitely important to qualify what sort of "material design elements" you talk about then. ;) I'd definitely agree shadows can help define layers of the UI. (Though it can't be the sole indicator of hierarchy, as fine details like shadows are completely overlooked by some viewers.) There are other elements I'd be more wary of though, like Google's recent fascination with animations, which can be resource-intensive.

**Comment by TheElf** [comment]
Lol animations are so pretty though. But I definitely see your point. I
think a lot of the animations would be fine if they're implemented as pure
CSS (yes I know some of them can't be implemented this way). This means it
will take advantage of graphics acceleration on supported platforms, and
can be disabled very easily.

But believe me, I think all of the icons and buttons should have clear
labeling and be linked to reasonable actions

**Comment by lifehome** [comment]
@fix Hmm, the title font of the first pic... I feel Geocities and Comic Sans are coming after me...

**Comment by MartinDesigner** [comment]
![imghome](https://cloud.githubusercontent.com/assets/13809960/9289329/0bef602c-436c-11e5-80fc-3ec07f82dd8d.jpg)
So, as a supplement, however as a main point try this amazing free app. Createer.com is an online software for easily designing online ads, responsive landing pages and print materials such as flyers, posters, covers, invitations and more. Createer lets you easily share your designs on social media, send them via email, download as HTML, image or PDF. You can even design UI mockups for mobile as for standard display (html, js, css always generating without any coding required from you). Createer also includes 3,000 predesigned templates (browse here: https://beta2.createer.com/Templates?ghi) and over 250,000 images (illustrations, photos and icons) to work with in its user-friendly drag’n’drop enviroment. And finally this amazing stuff is here: https://beta2.createer.com/?ghi

**Comment by TiagoFreitas87** [comment]
For pomodoro timers, check out [Pomello](http://www.tinynudge.co/pomello).

**Comment by almereyda** [comment]
@SimonSarazin Nice to see you here. Is pierreozoux also around?

**Comment by pierreozoux** [comment]
Thanks @almereyda to let me know the project was here :) I was still on the gitlab :/ Great to see new development!

*excited*
